bin/mkosi

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash
+
+PYTHON=$(command -v python3)
+PYVERSION=$($PYTHON -c '
+import sys
+version = sys.version_info
+print(f"{version[0]}.{version[1]}")
+')
+OLD_PYTHONPATH="$PYTHONPATH"
```

Comment:
Isn't it simpler to simply use a different variable name for the extra python path additions instead of saving the old pythonpath just so we can reuse PYTHONPATH as a variable name?

Reply:
I don't think that makes much of a difference, since we'd still need to check at the end whether we need to combine the two or not, but I have no particular preference either way. I think I might have chosen this way, because it is similar to the dance `bin/activate` does in a virtualenv.